### PR TITLE
Fixed some bugs to FC host-based replication

### PR DESCRIPTION
**Pull request by xxwjj**
**What this PR does / why we need it**:
1. Fixed some bugs about replication so that make it supporting FC host-based replication.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: contrib/connector/fc/helper.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+
+package fc
+
+import "strings"
```
Comment (leonwanghui) [inline contrib/connector/fc/helper.go]: Can you update this declaration?

File: pkg/dock/discovery/discovery.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ import (
 	. "github.com/opensds/opensds/pkg/utils/config"
 	"github.com/satori/go.uuid"
 	"strings"
+	"github.com/opensds/opensds/contrib/connector/fc"
```
Comment (leonwanghui) [inline pkg/dock/discovery/discovery.go]: Can you use `go fmt` to update the code style?

File: contrib/connector/fc/helper.go (hunk 1 above)
Comment (xxwjj) [reply contrib/connector/fc/helper.go]: already go fmt

**Comment by leonwanghui** [comment]
Is this patch finished or WIP?

**Comment by xing-yang** [comment]
@xxwjj This looks fine to me.  Are you still working on something?

**Comment by xing-yang** [comment]
Need a rebase.

**Comment by xxwjj** [comment]
@xing-yang Everthing has been done, please merge it, thanks.

**Pull request merged**